types/module/module.go

```diff
@@ -341,6 +345,19 @@ func (m *Manager) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) map[string
 	return genesisData
 }
 
+// checkForgottenModules checks that we didn't forget any modules in the
+// SetOrder* functions.
+func (m *Manager) checkForgottenModules(setOrderFnName string, moduleNames ...string) {
```

Comment:
what's the goal of using varargs, rather than slice? Instead of passing `moduleNames...` we can just pass `moduleNames` in line 274 and use `[]string` here.